src/absil/ilreflect.fs

```diff
@@ -336,11 +336,11 @@ let convTypeRefAux (cenv:cenv) (tref:ILTypeRef) =
             | None ->
                 let asmName    = convAssemblyRef asmref
                 FileSystem.AssemblyLoad(asmName)
-        let typT       = assembly.GetType(qualifiedName)
+        let typT = assembly.GetType(qualifiedName, throwOnError=true)
         typT |> nonNull "convTypeRefAux" 
     | ILScopeRef.Module _ 
     | ILScopeRef.Local _ ->
-        let typT = Type.GetType(qualifiedName,true) 
+        let typT = Type.GetType(qualifiedName, throwOnError=true) 
         typT |> nonNull "convTypeRefAux" 
```

Comment:
Do you still need the nullcheck when using : Type.GetType(qualifiedName, throwOnError=true)

https://msdn.microsoft.com/en-us/library/c5cf8k43(v=vs.110).aspx